ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ServiceToOpenAPIMapper.java

```diff
@@ -222,6 +233,24 @@ public static OASResult generateOAS(OASGenerationMetaInfo oasGenerationMetaInfo)
         }
     }
 
+    private static boolean isInterceptableService(ServiceDeclarationNode serviceDefinition,
+                                                  SemanticModel semanticModel) {
+        AtomicBoolean isInterceptable = new AtomicBoolean(false);
+        semanticModel.symbol(serviceDefinition).ifPresent(symbol -> {
+            if (symbol instanceof ServiceDeclarationSymbol serviceSymbol) {
+                serviceSymbol.typeDescriptor().ifPresent(serviceType ->
+                    semanticModel.types().getTypeByName("ballerina", "http", "",
```

Comment:
Shall we use constants for `ballerina` and `http`

Reply:
+1

Reply:
Fixed via [b7b157e](https://github.com/ballerina-platform/openapi-tools/pull/1614/commits/b7b157e80d2574a5f1708c915c92dad05bb532d2)